common/util.c

```diff
@@ -123,6 +123,21 @@ uint32_t parse_color(const char *color) {
 	return res;
 }
 
+bool parse_boolean(const char *boolean, bool current) {
```

Comment:
Do we want to handle errors? (ie. `bool parse_boolean(const char *str, bool current, bool *out)`)

Reply:
I thought this too, but it looks like the function returns false for invalid values and if that matches i3's behavior we should keep it.

Reply:
It would make sense to add a comment stating that.

Reply:
> It would make sense to add a comment stating that.

Added

> What about case sensitivity? Shouldn't these use `strcasecmp`?

Fixed